### PR TITLE
Add `currentTime` method

### PR DESCRIPTION
**Pull request by igorkamyshev**
### Description

Discussed in #152

### Checklist for a new method

- [x] Create directory for new method in root in `param-case`
- [x] Place code to `{method-name}/index.js` in CommonJS export in `camelCase` named export
- [x] Add types to `{method-name}/index.d.ts`
- [x] Add tests to `{method-name}/{method-name}.test.ts`
- [x] Add **fork** tests to `{method-name}/{method-name}.fork.test.ts`
- [x] Add **type** tests to `test-typings/{method-name}.ts`
  - Use `// @ts-expect-error` to mark expected type error
  - `import { expectType } from 'tsd'` to check expected return type
- [x] Add documentation in `{method-name}/reade.md`
  - Add header `Patronum/MethodName`
  - Add section with overloads, if have
  - Add `Motivation`, `Formulae`, `Arguments` and `Return` sections for each overload
  - Add useful examples in `Example` section for each overload
- [x] Add section to `README.md` in root
  - Add method to table of contents `- [MethodName](#methodname)`
  - Add section `## [MethodName](/method-name 'Documentation')`
  - With summary and simple example

### CODE REVIEW

**Comment by igorkamyshev** [comment]
@sergeysova hello. Could you review this PR please?

File: babel-plugin-factories.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "patronum",
     "patronum/combine-events",
     "patronum/condition",
+    "patronum/currentTime",
```
Comment (sergeysova) [inline babel-plugin-factories.json:6]: Please, run `yarn build:methods` and commit changes

File: current-time/current-time.test.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    stop();
+
+    // 11 = 10 updates + 1 update on start
+    expect(watcher).toBeCalledTimes(11);
+  });
+
+  test('should use custom interval', () => {
+    const testIntrevals = [20, 200, 2000];
+
+    for (const interval of testIntrevals) {
```
Comment (sergeysova) [inline current-time/current-time.test.ts:118]: You can use [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) to simplify test:

```suggestion
  test.each([20, 200, 2000])('should use %i interval', (interval) => {
```

File: current-time/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const { createEffect, restore, sample } = require('effector');
+
+function currentTime({ start, stop, interval = 100 }) {
```
Comment (sergeysova) [inline current-time/index.js:3]: What about adding a `clock` argument?

```ts
const $time = time({ start, stop, clock });

$time.getState() // time at initialize moment

clock() // now, $time changed to moment when clock triggered
```

**Review by sergeysova — CHANGES_REQUESTED** [review]

**Comment by igorkamyshev** [comment]
So, I'm closing this PR. I'll send another one with `interval` implementation, and then I'll create `time` method based on `interval`. Thanks for review and ideas.

**Pull request closed without merging**